continuous_integration/hdfs/install.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -xe
 
-docker exec hdfs conda install -y -q dask pyarrow">=0.14.0,!=2.0.0" fsspec pyyaml openjdk -c conda-forge
+docker exec hdfs conda install -y -q dask pyarrow">=0.14.0,!=2.0.0" fsspec pyyaml">=5.3.1" openjdk -c conda-forge
```

Comment:
@jrbourbeau @jsignell I wasn't sure if this needed to be changed or not

Reply:
I can't see how it would hurt :)